events.go

```diff
@@ -133,6 +133,9 @@ func (s *Session) handleNodeEvent(frames []frame) {
 		// TODO: can we be sure the order of events in the buffer is correct?
 		switch f := frame.(type) {
 		case *topologyChangeEventFrame:
+			host, port := s.cfg.translateAddress(f.host.String(), f.port)
+			f.host = net.ParseIP(host)
```

Comment:
events are defined as having a resolved IP address, if a user returns something which is not an IP address we will run into issues later on.